src/iconsManifest/languages.ts

```diff
@@ -286,6 +286,7 @@ export const languages: ILanguageCollection = {
   typescript: { ids: 'typescript', defaultExtension: 'ts' },
   typescriptreact: { ids: 'typescriptreact', defaultExtension: 'tsx' },
   typo3: { ids: 'typoscript', defaultExtension: 'typoscript' },
+  vapor: { ids: 'vapor', defaultExtension: 'vapor.yml' },
```

Comment:
See my comment about the necessity of using a `language` here. I've searched the marketplace and found out about one extension. The id, though, it's not `vapor` but `vaporlang`. You can check it out [here](https://github.com/minkcv/vaporlang-vscode/blob/master/package.json#L16).

Reply:
No, it's Laravel's Vapor (serverless platform & framework), sorry I didn't describe it on the PR clearly. So it doesn't have an extension (yet), but it'll be cool if it has its own icon as the file is always called vapor.yml.